browser/src/app/SlideshowMiddleware.ts

```diff
@@ -14,8 +14,15 @@
 
 // SlideBitmapManager handles the layers for the slideshow
 // It provides utility to decompress the row data from zstd and then make bitmaps from it
+interface CompressedSlideLayer {
+	json: any;
```

Comment:
could we have in some followup a real typing here?
helps to catch bugs later

Reply:
yep, we can do an interface with optional fields, at leas it gives use some type checking